ansible/configs/osp-migration/infra.yml

```diff
@@ -46,7 +46,7 @@
         parameters:
           project_name: "{{ osp_project_name }}"
           project_guid: "{{ guid }}"
-          project_description: "created:{{ ansible_date_time.epoch }}"
+          project_description: "created:{{ ansible_date_time.epoch | default('') }}"
```

Comment:
This should not need a default... If `ansible_date_time` is not set, then the `setup` module (or `gather_facts`) should be run.